m2cgen/interpreters/r/linear_algebra.r

```diff
@@ -0,0 +1,6 @@
+add_vectors <- function(v1, v2) {
```

Comment:
I wonder why do we need this overhead together with the `LinearAlgebraMixin`, if we can just easily override `interpret_bin_vector_expr` and `interpret_bin_vector_num_expr` methods in place to provide these one-line expressions. Similarly to what we do in Python interpreter (with numpy). Am I missing something?